app/assets/javascripts/backbone/views/loan_questions_view.coffee

```diff
@@ -26,7 +26,8 @@ class MS.Views.LoanQuestionsView extends Backbone.View
     'tree.move .jqtree': 'moveNode'
     'click .delete-action': 'confirmDelete'
     'confirm:complete .delete-action': 'deleteNode'
-    'change input[name="custom_field[override_associations]"]': 'showHideAssociations'
+    'change [name="custom_field[override_associations]"]': 'showHideAssociations'
```

Comment:
We currently have this set to change. Yet when the modal is first open for a specific loan question, if override parent exists in the form and is set to inherit from parent, the table needs to be hidden. If set to override parent, the table needs to be visible.